evm/gas-service.js

```diff
@@ -304,24 +314,42 @@ async function processCommand(config, chain, options) {
             }
 
             try {
-                const tx = await timeout(
-                    gasService.updateGasInfo(chainsToUpdate, gasInfoUpdates, gasOptions),
-                    chain.timeout || 60000,
-                    new Error(`Timeout updating gas info for ${chain.name}`),
-                );
-
-                printInfo('TX', tx.hash);
-
-                const receipt = await timeout(
-                    tx.wait(chain.confirmations),
-                    chain.timeout || 60000,
-                    new Error(`Timeout updating gas info for ${chain.name}`),
-                );
-
-                const eventEmitted = wasEventEmitted(receipt, gasService, 'GasInfoUpdated');
-
-                if (!eventEmitted) {
-                    printWarn('Event not emitted in receipt.');
+                if (options.useRelay) {
+                    const gasLimit = Number(await gasService.estimateGas.updateGasInfo(chainsToUpdate, gasInfoUpdates));
+
+                    const tx = await timeout(
```

Comment:
This API is asynchronous I believe, so we don't need a timeout. you can double check

Reply:
yep, it should return quickly and do relaying in the background. It also won't return a receipt so `tx.hash` will be `undefined`. The only field it returns is `relay_id`